framework/components/blockchain/aptos.go

```diff
@@ -21,9 +21,8 @@ func defaultAptos(in *Input) {
 	if in.Image == "" {
 		in.Image = "aptoslabs/tools:aptos-node-v1.27.2"
 	}
-	if in.Port != "" {
-		framework.L.Warn().Msg("'port' field is set but only default port can be used: 8080")
-	}
+	framework.L.Warn().Msg("Aptos node API can only be exposed on port 8080!")
```

Comment:
curious: why?

Reply:
Not exposed in their CLI, can't override that port